software/safeline.yml

```
@@ -0,0 +1,11 @@
+name: "SafeLine"
+website_url: "https://waf.chaitin.com/"
+source_code_url: "https://github.com/chaitin/SafeLine"
+description: "WAF (Web Application Firewall) / reverse proxy to protect your web apps from attacks and exploits."
```

I dont think you need WAF and then explain what it is. I think this is cleaner.

```suggestion
description: "Web application firewall / reverse proxy to protect your web apps from attacks and exploits."
```